cinema/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(required=True)
+    duration = serializers.IntegerField(required=True)

```

Comment:
`required=True` is used by default in [`serializers.Serializer` ](https://www.django-rest-framework.org/api-guide/fields/#:~:text=normally%20an%20error%20will%20be%20raised%20if%20a%20field%20is%20not%20supplied%20during%20deserialization.%20set%20to%20false%20if%20this%20field%20is%20not%20required%20to%20be%20present%20during%20deserialization.l)

Reply:
agree, https://www.django-rest-framework.org/api-guide/fields/#required